app/models/source_file.rb

```diff
@@ -23,14 +23,12 @@ class SourceFile < ActiveRecord::Base
   after_destroy { s3_object.try(:delete) }
 
   def to_jq_upload
-    {
-      'id' => id,
+    { 'id' => id,
       'name' => file_name,
       'size' => file_size,
       'url' => url,
       'image' => self.is_image?,
-      'delete_url' => Rails.application.routes.url_helpers.source_file_path(self, :format => :json)
-    }
+      'delete_url' => Rails.application.routes.url_helpers.source_file_path(self, :format => :json) }
```

Comment:
it looks cleaner to leave it as it was here - one line per option as well as one per opening and closing of the hash